src/screens/VisualTests/SnapshotControls.tsx

```diff
@@ -330,7 +323,8 @@ export const SnapshotControls = ({
             <ActionButton
               containsIcon
               aria-label={isOutdated ? "Run new tests" : "Rerun tests"}
-              onClick={() => startDevBuild()}
+              onClick={startBuild}
+              disabled={isRunning}
```

Comment:
This might be too picky, but it would be nice to also capture this in a story

Reply:
It is, in [Pending Local Build Starting](https://www.chromatic.com/component?appId=6480e1b0042842f149cfd74c&csfId=screens-visualtests-visualtests--pending-local-build-starting&buildNumber=1829&k=65e893ff088ac5e6bef59916-Bottom+2%252Dup-interactive-true&h=15&b=-8) for example.